app/controllers/contact_infos_controller.rb

```diff
@@ -41,7 +41,7 @@ def resend_confirmation
                   path = current_user.is_email_pending? ? verification_sent_path :
                                                           profile_path(active_tab: :email)
```

Comment:
After `ContactInfoResendConfirmation` runs, when would `current_user.is_email_pending?` be false?